arroyo-worker/src/connectors/kafka/sink/test.rs

```diff
@@ -30,22 +30,40 @@ impl KafkaTopicTester {
             .set("group.id", format!("{}-{}-creator", job_id, "operator_id"))
             .create()
             .unwrap();
-        admin_client
-            .delete_topics(&[&self.topic], &AdminOptions::new())
-            .await
-            .expect("deletion should have worked");
-        tokio::time::sleep(Duration::from_secs(1)).await;
-        admin_client
-            .create_topics(
-                [&NewTopic::new(
-                    &self.topic,
-                    num_partitions,
-                    rdkafka::admin::TopicReplication::Fixed(1),
-                )],
-                &AdminOptions::new(),
-            )
-            .await
-            .expect("new topic should be present");
+        let mut tries = 0;
+        while tries < 5 {
+            let delete_result = &admin_client
+                .delete_topics(&[&self.topic], &AdminOptions::new())
+                .await
+                .expect("deletion should have worked")[0];
+            tokio::time::sleep(Duration::from_secs(1)).await;
```

Comment:
this always adds a second, maybe move this into the retry handler?

Reply:
This sleep had already been in there, I think because of some issues with the delete taking some time to propogate on the kafka side of things.